CHANGELOG.md

```diff
@@ -2,12 +2,17 @@
 
 All notable changes to the "azure-cosmosdb" extension will be documented in this file.
 
-## Unreleased
+## 0.16.0 - 2021-02-23
 ### Added
 - Now depends on the "Azure Resources" extension, which provides a "Resource Groups" and "Help and Feedback" view
 
 ### Changed
 - "Report an Issue" button was removed from errors. Use the "Help and Feedback" view or command palette instead
+- Grouped items in all context menus to improve access and readability
```

Comment:
Here is how I view each section of the changelog:

- Added: new features or enhancements
- Changed: Removed an old feature/enhancement, but replaced it with something else
- Removed: Removed an old feature/enhancement with no replacement

I think both of your bullets would fit better in "Added"